src/cairoUtilFuncGen/memory/implicitConversion.ts

```diff
@@ -425,13 +425,21 @@ export class MemoryImplicitConversionGen extends StringIndexedFuncGen {
   }
 }
 
-function getBaseType(type: TypeNode): TypeNode {
+export function getBaseType(type: TypeNode): TypeNode {
   const deferencedType = generalizeType(type)[0];
   return deferencedType instanceof ArrayType
     ? getBaseType(deferencedType.elementT)
     : deferencedType;
 }
 
+export function getNestedNumber(type: TypeNode): string {
```

Comment:
This isn't used in this file anymore. Could you move it to the file where it is. Also I don't think it should be called something else given that it's not getting a number